.gitignore

```diff
@@ -1,52 +1,43 @@
-# Compiled class file
```

Comment:
Should this be committed? I tend to think this should be maintained by each developer separately?

Reply:
I generally keep the gitignore file really small. In this case, it would include target and not much else.

Developer specific environment entries should go in .git/config/info/exclude

IOW, why pick IntelliJ, Netbeans and VScode? And not other popular IDEs like vim?